layouts/account.tsx

```diff
@@ -134,6 +164,26 @@ const AccountLayout = ({
     setSelectedStakingAction("delegate");
   }, [setSelectedStakingAction]);
 
+  /* PART OF https://github.com/livepeer/explorer/pull/427 - REMOVE ONCE SERVER-SIDE ISSUE IS FIXED */
```

Comment:
Let's mark this as `TODO: REMOVE ONCE SERVER-SIDE ISSUE IS FIXED` as that shows up in your editor under todos.